cmake/test.cmake

```diff
@@ -1,20 +1,22 @@
 set(MAYA_USD_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 
 if(MayaUsd_FOUND)
-    #Add MAYAUSD_MOD_PATH (the path where maya USD .mod file is) to the MAYA_MODULE_PATH
-    # Get the current value of the environment variable
-    set(CURRENT_MAYA_MODULE_PATH $ENV{MAYA_MODULE_PATH})
-    # Append the new path to the current value
-    if(IS_MACOSX OR IS_LINUX) 
-        #Linux and OSX
-        set(MAYA_MODULE_PATH "${CURRENT_MAYA_MODULE_PATH}:${MAYAUSD_MOD_PATH}")
-    else()
-        #Windows
-        set(MAYA_MODULE_PATH "${CURRENT_MAYA_MODULE_PATH}\;${MAYAUSD_MOD_PATH}")
+    if (MAYAUSD_MOD_PATH)
```

Comment:
The only change is is the testing of MAYAUSD_MOD_PATH before this was not done as MayaUsd_FOUND would have been false if MAYAUSD_MOD_PATH was not found.

Reply:
You could have also done `if(MayaUsd_FOUND AND MAYAUSD_MOD_PATH)`